j2v/generation/generator.py

```diff
@@ -100,6 +100,7 @@ def __add_explore_join(self, new_view_name, current_view, key, current_path):
         :return:
         """
 
+        key = key.lower()
```

Comment:
You cannnot change anything in the data path, as it's case-sensitive, see the docs:
https://docs.snowflake.net/manuals/user-guide/querying-semistructured.html#traversing-semi-structured-data